.github/workflows/docs.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         sudo apt-get update --fix-missing
         sudo apt-get autoremove
         sudo apt-get autoclean
-        pip install tox
+        pip install tox==3.27.1
```

Comment:
No it is not, read PR description

Reply:
It's obvious that the last successful run (unpinned) would be a higher version (highest at the time). But the correct pin is the one to our tomte version. So that we have consistency with the Pipfile (and local runs). Ideally it should be pinned as such: `tomte[tox]==0.1.5`